web/src/main/java/com/bazaarvoice/emodb/web/scanner/ScanOptions.java

```diff
@@ -54,8 +52,7 @@ private ScanOptions(@JsonProperty ("placements") List<String> placements,
                         @JsonProperty ("scanByAZ") Boolean scanByAZ,
                         @JsonProperty ("maxConcurrentSubRangeScans") Integer maxConcurrentSubRangeScans,
                         @JsonProperty ("rangeScanSplitSize") Integer rangeScanSplitSize,
-                        @JsonProperty ("maxRangeScanTime") Long maxRangeScanTime,
-                        @JsonProperty ("compactionEnabled") Boolean compactionEnabled) {
+                        @JsonProperty ("maxRangeScanTime") Long maxRangeScanTime) {
```

Comment:
Since we're removing a field can you add the following annotation to the class:

`@JsonIgnoreProperties(ignoreUnknown = true)`

This way when it tries to read existing stash records from the Emo table it won't choke on old records where this attribute is present.